cmd/build.go

```diff
@@ -222,41 +307,67 @@ func queueMostRecentReleasesForChain(
 	return nil
 }
 
+func loadChainsYaml(configFile string) error {
+	if _, err := os.Stat(configFile); err != nil {
+		return fmt.Errorf("error checking for file: %s: %w", configFile, err)
+	}
+	bz, err := os.ReadFile(configFile)
+	if err != nil {
+		return fmt.Errorf("error reading file: %s: %w", configFile, err)
+	}
+	var newChains []ChainNodeConfig
+	err = yaml.Unmarshal(bz, &newChains)
+	if err != nil {
+		return fmt.Errorf("error unmarshalling yaml from file: %s: %w", configFile, err)
+	}
+	chains = newChains
+	return nil
+}
+
 var buildCmd = &cobra.Command{
 	Use:   "build",
 	Short: "Build the docker images",
 	Long: `By default, fetch the last 5 releases in the repositories specified in chains.yaml.
 For each tag that doesn't exist in the specified container repository,
 it will be built and pushed`,
 	Run: func(cmd *cobra.Command, args []string) {
-		containerRegistry, _ := cmd.Flags().GetString("registry")
-		fmt.Printf("Container registry: %s\n", containerRegistry)
-
 		cmdFlags := cmd.Flags()
 
+		configFile, _ := cmdFlags.GetString("file")
+		if configFile == "" {
+			// try to load a local chains.yaml, but do not panic for any error, will fall back to embedded chains.
+			cwd, err := os.Getwd()
+			if err == nil {
+				chainsYamlSearchPath := filepath.Join(cwd, "chains.yaml")
+				if err := loadChainsYaml(chainsYamlSearchPath); err != nil {
+					fmt.Printf("No config found at %s, using embedded chains. pass -f to configure config.yaml path.\n", chainsYamlSearchPath)
```

Comment:
I'm getting tripped up by the word "embedded" here. It still must be a local file on your filesystem? 

Or have we thought about using `go:embed` to embed the yaml into the binary, so no local files needed?

Reply:
Thanks for that clarification. I missed it.